moveit_kinematics/cached_ik_kinematics_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+find_package(Boost COMPONENTS filesystem program_options REQUIRED)
+
+set(MOVEIT_LIB_NAME moveit_cached_ik_kinematics_base)
+add_library(${MOVEIT_LIB_NAME} src/ik_cache.cpp)
+set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+target_link_libraries(${MOVEIT_LIB_NAME}
+    ${Boost_FILESYSTEM_LIBRARY}
+    ${catkin_LIBRARIES})
+install(TARGETS ${MOVEIT_LIB_NAME} LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+
+
+set(MOVEIT_LIB_NAME moveit_cached_ik_kinematics_plugin)
+add_library(${MOVEIT_LIB_NAME} src/cached_ik_kinematics_plugin.cpp)
+set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+target_link_libraries(${MOVEIT_LIB_NAME}
+    moveit_cached_ik_kinematics_base
+    moveit_kdl_kinematics_plugin
+    moveit_srv_kinematics_plugin
+    ${OMPL_LIBRARIES}
+    ${catkin_LIBRARIES})
+if(trac_ik_kinematics_plugin_FOUND)
+    target_link_libraries(${MOVEIT_LIB_NAME} trac_ik_kinematics_plugin)
+    set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES COMPILE_DEFINITIONS "CACHED_IK_KINEMATICS_TRAC_IK")
+endif()
+install(TARGETS ${MOVEIT_LIB_NAME} LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+
+# This is just for testing purposes; the arms from Universal Robots have
+# analytic solvers, so caching just adds extra overhead.
+find_package(ur_kinematics QUIET)
+if(ur_kinematics_FOUND)
+    include_directories(${ur_kinematics_INCLUDE_DIRS})
+    foreach(UR 3 5 10)
+        set(MOVEIT_LIB_NAME moveit_cached_ur${UR}_kinematics_plugin)
+        add_library(${MOVEIT_LIB_NAME} src/cached_ur_kinematics_plugin.cpp)
+        set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
+        find_library(ur${UR}_pluginlib ur${UR}_moveit_plugin PATHS ${ur_kinematics_LIBRARY_DIRS})
+        target_link_libraries(${MOVEIT_LIB_NAME}
+            moveit_cached_ik_kinematics_base
+            ${ur${UR}_pluginlib}
+            ${OMPL_LIBRARIES}
+            ${catkin_LIBRARIES})
+        install(TARGETS ${MOVEIT_LIB_NAME} LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+    endforeach()
+endif()
+
+# not sure why this is necessary: it should already be part of ${catkin_LIBRARIES}
+find_library(MOVEIT_ROBOT_MODEL_LOADER_LIBRARY moveit_robot_model_loader)
```

Comment:
This does not make sense - the [moveit_robot_model_loader](https://github.com/ros-planning/moveit/blob/091f766175fadb1fd4cdab405715237cfdc88f80/moveit_ros/planning/planning_components_tools/CMakeLists.txt) is located in ``moveit_ros_planning``, which is already a dependency of this package. Can you investigate further?